components/org.wso2telco.analytics.sparkUdf/src/main/java/org/wso2telco/analytics/sparkUdf/service/BillUpdaterService.java

```diff
@@ -1,89 +1,93 @@
 package org.wso2telco.analytics.sparkUdf.service;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS00120)